src/main/java/org/folio/auth/authtokenmodule/MissingAlgorithmException.java

```diff
@@ -0,0 +1,7 @@
+package org.folio.auth.authtokenmodule;
+
+public class MissingAlgorithmException extends Exception {
```

Comment:
Can `Exception` been replaced by `RuntimeException` or `Error` to make it an unchecked exception because we cannot recover from it?